rules/linux/execution_shell_evasion_linux_binary.toml

```diff
@@ -0,0 +1,107 @@
+    "https://gtfobins.github.io/gtfobins/find/",
+    "https://gtfobins.github.io/gtfobins/flock/",
+    "https://gtfobins.github.io/gtfobins/gcc/",
+    "https://gtfobins.github.io/gtfobins/mysql/",
+    "https://gtfobins.github.io/gtfobins/nice/",
+    "https://gtfobins.github.io/gtfobins/ssh/",
+    "https://gtfobins.github.io/gtfobins/vi/",
+    "https://gtfobins.github.io/gtfobins/capsh/",
+    "https://gtfobins.github.io/gtfobins/byebug/",
+]
+risk_score = 47
+rule_id = "52376a86-ee86-4967-97ae-1a05f55816f0"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Execution", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+ (
+   (process.name in ("bash", "sh", "dash"," ash") and process.parent.name in ("byebug")) or
+
+   (process.name == "sensible-pager" and process.args in ("/bin/sh", "/bin/bash", "/bin/dash", "sh", "bash", "dash") and process.parent.name in ("apt", "apt-get") and process.parent.args == "changelog") or
+
+   (process.name in ("sh", "bash", "dash") and process.parent.name in ("nawk", "mawk", "awk", "gawk") and process.parent.args : "BEGIN {system(*)}") or
+
+   (process.name == "busybox" and process.args_count == 2 and process.args in ("/bin/sh", "/bin/ash", "sh", "ash")) or
+
+   (process.name in ("sh", "dash", "bash") and process.parent.name in ("c89","c99") and process.parent.args == "-wrapper" and process.parent.args in ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s")) or
+
+   (process.name == "capsh" and process.args == "--") or
+
+   (process.name in ("bash", "sh", "dash") and process.parent.name == "cpulimit" and process.parent.args == "-f" and process.parent.args in ("/bin/sh", "/bin/bash", "/bin/dash", "sh", "bash", "dash")) or
+
+   (process.parent.name == "crash" and process.parent.args == "-h" and process.name == "sh") or
+
+   (process.name == "env" and process.args_count == 2 and process.args in ("/bin/sh", "/bin/bash", "sh", "bash")) or
+
+   (process.name in ("bash", "sh", "dash") and process.parent.name == "expect" and process.parent.args == "-c" and process.parent.args in ("spawn /bin/sh;interact", "spawn /bin/bash;interact", "spawn /bin/dash;interact", "spawn sh;interact", "spawn bash;interact", "spawn dash;interact")) or
+
+   (process.name in ("bash", "sh") and process.parent.name == "find" and process.parent.args == "-exec" and process.parent.args == ";" and process.parent.args in ("/bin/bash", "/bin/sh", "bash", "sh")) or
+
+   (process.parent.name == "flock" and process.parent.args == "-u" and process.parent.args == "/" and process.parent.args in ("/bin/sh", "/bin/bash", "/bin/dash", "sh", "bash", "dash") and process.name in ("bash", "dash","sh")) or
+
+   (process.name in ("sh", "dash", "bash")  and process.parent.name == "gcc" and process.parent.args == "-wrapper" and process.parent.args in ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s")) or
```

Comment:
can flatten with c99 since args are identical

Reply:
Will make this changes.